jinja2/nativetypes.py

```diff
@@ -20,10 +21,13 @@ def native_concat(nodes):
     if not head:
         return None
 
+    if isinstance(nodes, types.GeneratorType):
```

Comment:
`collections.Generator` is probably more correct here.

Reply:
Looks like `collections.abc.Generator` [1] is new in Python 3.5 though. We'd need to use `types.GeneratorType` for previous versions?

[1] https://docs.python.org/3/library/collections.abc.html#collections.abc.Generator